configuration/coding_plans.py

```diff
@@ -68,6 +68,8 @@ def get_rqa_coding_plans(pipeline_name):
                    run_id_field="rqa_s01e03_run_id",
                    coda_filename="UNICEF_COVID19_SOM_s01e03.json",
                    icr_filename="rqa_s01e03.csv",
+                   katikati_survey_start_time="2020-07-24T14:00:00+03:00",
```

Comment:
I think this should be a datetime rather than a string since it's python rather than json